src/AnswersCore.ts

```diff
@@ -18,7 +18,6 @@ import { AutocompleteService } from './services/AutocompleteService';
  * @public
  */
 export class AnswersCore {
-  /** @internal */
```

Comment:
for my edification was it already possible to instantiate your own AnswersCore, since we were already exporting the class, or did this annotation have say api-extractor prevent people from doing so via typescript yelling at them?

Reply:
The constructor was exported but it took no arguments. I think it exported the "default"/"no params" constructor if that makes sense?